src/lightning_lite/plugins/__init__.py

```diff
@@ -0,0 +1,17 @@
+from lightning_lite.plugins.precision.mixed import MixedPrecisionPlugin
+from pytorch_lightning.plugins.precision.deepspeed import DeepSpeedPrecisionPlugin
+from pytorch_lightning.plugins.precision.double import DoublePrecisionPlugin
+from pytorch_lightning.plugins.precision.native_amp import NativeMixedPrecisionPlugin
+from pytorch_lightning.plugins.precision.precision_plugin import PrecisionPlugin
+from pytorch_lightning.plugins.precision.tpu import TPUPrecisionPlugin
+from pytorch_lightning.plugins.precision.tpu_bf16 import TPUBf16PrecisionPlugin
+
+__all__ = [
+    "DeepSpeedPrecisionPlugin",
+    "DoublePrecisionPlugin",
+    "MixedPrecisionPlugin",
+    "NativeMixedPrecisionPlugin",
+    "PrecisionPlugin",
+    "TPUPrecisionPlugin",
+    "TPUBf16PrecisionPlugin",
```

Comment:
Reviewer vote:

👍 Remove the Plugin postfix from the names
👎 Keep as is

Reply:
We chose the suffix because of the `plugins=` argument in the Trainer and because there was a shared Plugin interface.

Neither is true for Lite, so :+1:

Reply:
We got votes from code owners Justus, Thomas, Carlos and I vote for this as well. So far no concerns so I'm going to push a commit with this change.